packages/client/hmi-client/src/workflow/ops/model-comparison/tera-model-comparison.vue

```diff
@@ -108,7 +108,8 @@ const cellWidth = computed(() => `${85 / modelsToCompare.value.length}vw`);
 
 const kernelManager = new KernelSessionManager();
 
-async function addModelForComparison(modelId: string) {
+async function addModelForComparison(modelId: string | undefined | null) {
```

Comment:
```suggestion
async function addModelForComparison(modelId: Model["id"] | undefined | null) {
```

Reply:
The method signature should just be `string` or `Mode["id"`, it seems to defeat the purpose when we claim to  accepts null or undefined, only to immediate reject it via `if (!modelId) return;`

Reply:
I agree, this was done during the hackathon.
I'm trying to promote the use of our Types.ts as much as possible.